src/ReshowcaseUi.re

```diff
@@ -909,22 +878,12 @@ module DemoUnitFrame = {
       (),
     );
 
-  let useFullframeUrl: bool = [%mel.raw
-    {js|typeof USE_FULL_IFRAME_URL === "boolean" ? USE_FULL_IFRAME_URL : false|js}
-  ];
```

Comment:
Is this feature removed?

Reply:
my bad. I was not sure what it was doing and how to debug it. Will check